simnet_tests/README.md

```diff
@@ -0,0 +1,34 @@
+# Simulation tests, or high level integration tests.
```

Comment:
Also an idea: should we describe the content of simnet image with links to the corresponding repos somewhere? (We can do that in gitlab and provide here and in substrate copy the link to the description).

Reply:
yeah, I wanted to do that also. I waited for an answer from Ben, but then I forgot to add the description and links